src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2506,6 +2519,22 @@ private function registerHttpClientConfiguration(array $config, ContainerBuilder
         }
     }
 
+    private function registerThrottlingHttpClient(string $rateLimiter, string $name, ContainerBuilder $container): void
+    {
+        if (!class_exists(ThrottlingHttpClient::class)) {
+            throw new LogicException('Rate limiter support cannot be enabled as version 7.1+ of the HttpClient component is required.');
+        }
+
+        if (!interface_exists(LimiterInterface::class)) {
```

Comment:
this should actually check whether the rate_limiter is enabled in the FrameworkBundle config, not whether the component is installed, as installing it is not enough (no need to check again for the component being installed as enabling the rate_limiter without installing it is already checked when handling the rate_limiter config).
See `readConfigEnabled` for the checks about that.

Reply:
Done